lib/__tests__/findMatchingFiles-test.js

```diff
@@ -1,12 +1,14 @@
+/* global spyOn */
```

Comment:
Yeah, I was surprised too. I'm guessing it's because they want you to use mocking instead.